Gems/Atom/RPI/Code/CMakeLists.txt

```diff
@@ -6,6 +6,15 @@
 #
 #
 
+include(FetchContent)
```

Comment:
Adding @nick-l-o3de to provide guidance on if this is the correct way to pull in meshoptimizer. Would we instead want to pull in meshoptimer via 3p system? So we use FetchContent within https://github.com/o3de/3p-package-source, build the meshoptimizer 3p package for all platforms and pull that into RPI via that system

Reply:
I would actually prefer to avoid using the prebuilt 3p package system unless there's a good reason like it takes hours to build or something.

Reply:
Well, I do think meshoptimizer might be better put to 3p system since Meshlet gem uses it as well (it currently require users to manually compile and add the library tho)